projects/staffconnect.md

```diff
@@ -0,0 +1,16 @@
+---
+title: StaffConnect
+url: https://staffconnect-app.com
+author: Jeremy Jacob
+author_link: https://www.instagram.com/cool_runnings_11/
```

Comment:
Are you sure you would like to link your Instagram and not your GitHub or anything like that? Just double checking :)

Reply:
perhaps instagram is not the best.. will close this PR and make a new one with company page instead. Cheers!